test/README.md

```diff
@@ -22,33 +10,35 @@ go test $(go list ./... | grep -v third_party/)
 make test-unit
 ```
 
-By default `go test` will not run [the end to end tests](#end-to-end-tests),
-which need `-tags=e2e` to be enabled.
-
 ## End to end tests
 
-### prerequisite
+### Some prerequisites to run tests:
+
+1. To run end to end tests, you will need to have a running Tekton pipeline deployed on your cluster, see [Install Pipeline](../DEVELOPMENT.md#install-pipeline).
+
+2. Next install Tekton Triggers in the cluster. Depending on which version of Triggers you want to install run one of the commands from [Installation](https://github.com/tektoncd/triggers/blob/main/docs/install.md#installation)
+
+Set environment variable `TEST_CLUSTERTASK_LIST_EMPTY` to any value if tests are run in an environment which contains `clustertasks`. Set `SYSTEM_NAMESPACE` variable to appropriate value (say tekton-pipelines).The `SYSTEM_NAMESPACE` variable provides a configurable way to specify the namespace where Tekton Pipelines system components reside within the Kubernetes cluster.
+
+
+### Running
+
+End to end tests live in this directory. By default `go test` will not run [the end to end tests](#end-to-end-tests),
```

Comment:
>End to end tests live in this directory.

let's add a hyperlink to `this` keyword ?

Reply:
Sure @vinamra28 doing it.